spec/models/registration_spec.rb

```diff
@@ -3,6 +3,8 @@
 require 'spec_helper'
 
 describe 'Registration' do
+  let(:conference) { create(:conference) }
+  let(:registration1) { create(:registration, conference: conference, user: user) }
```

Comment:
thanks @ChrisBr :+1:  ...also, can i use some other tool along with rubocop to implement better code style practices?
meanwhile... working to make tests pass :)